tools/functional-tester/etcd-tester/lease_stresser.go

```diff
@@ -266,6 +266,12 @@ func (ls *leaseStresser) keepLeaseAlive(leaseID int64) {
 	defer ls.aliveWg.Done()
 	ctx, cancel := context.WithCancel(ls.ctx)
 	stream, err := ls.lc.LeaseKeepAlive(ctx)
+	defer func() {
```

Comment:
I don't think the CloseSend or cancel nil check is necessary (but the closure capturing the cancel is), can probably get away with `defer func() { cancel() }`